xqf.desktop.in

```diff
@@ -1,10 +1,10 @@
 [Desktop Entry]
-_Name=XQF Game Server Browser
-_GenericName=Game Server Browser
-_Comment=Locate and connect to game servers
+Name=XQF Game Server Browser
+GenericName=Game Server Browser
+Comment=Locate and connect to game servers
```

Comment:
This removes l10n support... 

On 18 juny de 2015 20:43:27 CEST, Thomas Debesse <notifications@github.com> wrote:
>> @@ -1,10 +1,10 @@
>>  [Desktop Entry]
>> -_Name=XQF Game Server Browser
>> -_GenericName=Game Server Browser
>> -_Comment=Locate and connect to game servers
>> +Name=XQF Game Server Browser
>> +GenericName=Game Server Browser
>> +Comment=Locate and connect to game servers
>
>what this modification does?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/XQF/xqf/pull/147/files#r32763993

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.